src/VSCodeExtension/src/formatter/operation-rules.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+'use strict';
+
+import { FormatRule } from "./index";
+
+/**
+ * Formats operation declarations.
+ * 
+ * @param code incoming code
+ * 
+ * @example `operation   Foo(q:Qubit, n   :Int)   :  Bool` ->
+ * `operation Foo (q : Qubit, n : Int) : Bool`
+ */
+const argsRule = (code: string): string => {
+    const operationMatcher: RegExp = /operation\s*(\w+)\s*\((.*)\)\s*:\s*(\w+)/g;
```

Comment:
What's a generic umbrella term for operations and functions?

Reply:
That's a good question, and I don't think there really is one. But if the intent of the file is for checking operation and function declarations, maybe a decent name would be declaration-rules.ts? Then it could also handle user-defined type declarations (`newtype` as documented [here](https://docs.microsoft.com/en-us/quantum/user-guide/language/types#user-defined-types)). That file name change can be made in following PRs, no need to gate this PR on the name change.

Reply:
FYI, we've generally used "callable" to mean "function or operation."